src/cmd-aws-replicate

```diff
@@ -63,7 +63,8 @@ def run_ore():
     if args.log_level:
         ore_args.extend(['--log-level', args.log_level])
     ore_args.extend(['aws', 'copy-image', '--image', source_image, '--region', source_region])
-    ore_args.extend(region_list)
+    for r in region_list:
+        ore_args.append(r)
```

Comment:
This is essentially what `extend` does: https://stackoverflow.com/questions/252703/what-is-the-difference-between-pythons-list-methods-append-and-extend

Reply:
:man_facepalming: